src/main/java/org/jabref/logic/ai/chatting/AiChatLogic.java

```diff
@@ -76,9 +76,7 @@ public AiChatLogic(AiPreferences aiPreferences,
         this.entries = entries;
         this.name = name;
         this.bibDatabaseContext = bibDatabaseContext;
-
```

Comment:
Revert all changes in this file.

You can do so in the command line:

    git checkout main -- src/main/java/org/jabref/logic/ai/chatting/AiChatLogic.java